src/lib/exec-fetcher.js

```diff
@@ -55,7 +34,7 @@ const isVersionOutdated = async ({ packageName, currentVersion }) => {
 const shouldFetchLatestVersion = async ({ binPath, packageName, execName, execArgs, pattern, log }) => {
   const execPath = path.join(binPath, getExecName({ execName }))
 
-  const exists = await execExist(execPath)
+  const exists = await isExe(execPath)
```

Comment:
`isexe` contains the code of `execExist()` not of `isExe()`.
See [source](https://github.com/isaacs/isexe/blob/master/mode.js)